app/code/Magento/WishlistGraphQl/etc/schema.graphqls

```diff
@@ -6,7 +6,16 @@ type Query {
 }
 
 type Customer {
-    wishlist: Wishlist! @resolver(class:"\\Magento\\WishlistGraphQl\\Model\\Resolver\\CustomerWishlistResolver") @doc(description: "Contains the contents of a customer's wish lists")  @cache(cacheable: false)
+    wishlists(
+        pageSize: Int = 20 @doc(description: "Specifies the maximum number of results to return at once. This attribute is optional."),
+        currentPage: Int = 1 @doc(description: "Specifies which page of results to return. The default value is 1.")
+    ): WishlistsOutput @doc(description: "All wishlists for customer.") @resolver(class:"\\Magento\\WishlistGraphQl\\Model\\Resolver\\CustomerWishlists")
+    wishlist: Wishlist! @deprecated(reason: "Use Customer.wishlists") @resolver(class:"\\Magento\\WishlistGraphQl\\Model\\Resolver\\CustomerWishlistResolver") @doc(description: "Contains the contents of a customer's wish lists")  @cache(cacheable: false)
+}
+
```

Comment:
Based on the architecture there is a need for querying specific variants of wishlist items (simple, configurable, virtual). I have also proposed minor updates to it after discussing with architects, which is about to be merged soon. Please find the changes here https://github.com/magento/architecture/pull/409.

The final schema will look like this https://github.com/magento/architecture/blob/df5a46ab0d5dfc006b37f2ca0f4de0853f04571d/design-documents/graph-ql/coverage/Wishlist.graphqls

<img width="696" alt="Screen Shot 2020-07-27 at 6 26 04 PM" src="https://user-images.githubusercontent.com/43612864/88601608-b0213880-d036-11ea-9f91-49c851a789ae.png">

We discussed about this yesterday and please add the coverage for it. Let me know if you have any questions. The rest looks good on the PR 👍

Reply:
Please cover this https://github.com/magento/partners-magento2ee/pull/248#discussion_r461231570 as a part of this PR.